draft-ietf-httpbis-http2.xml

```diff
@@ -995,64 +995,64 @@ HTTP2-Settings    = token68
               </t>
               <t>
-                If this state is reached as a result of sending a <x:ref>RST_STREAM</x:ref> frame,
-                the peer that receives the <x:ref>RST_STREAM</x:ref> might have already sent - or
-                enqueued for sending - frames on the stream that cannot be withdrawn.  An endpoint
-                MUST ignore frames that it receives on closed streams after it has sent a
-                <x:ref>RST_STREAM</x:ref> frame.  An endpoint MAY choose to limit the period over
-                which it ignores frames and treat frames that arrive after this time as being in
-                error.
+                Flow controlled frames (i.e., <x:ref>DATA</x:ref>) received in this state
+                are counted toward the connection flow control window, even though their content
+		is ignored as the sender will consider the frames to count
+                against the flow control window.
               </t>
               <t>
-                Flow controlled frames (i.e., <x:ref>DATA</x:ref>) received after sending
-                <x:ref>RST_STREAM</x:ref> are counted toward the connection flow control window.
-                Even though these frames might be ignored, because they are sent before the sender
-                receives the <x:ref>RST_STREAM</x:ref>, the sender will consider the frames to count
-                against the flow control window.
+                An endpoint might receive a <x:ref>PUSH_PROMISE</x:ref> in this state.  Therefore, a
+                <x:ref>RST_STREAM</x:ref> is needed to close any unwanted promised stream.
               </t>
               <t>
-                An endpoint might receive a <x:ref>PUSH_PROMISE</x:ref> frame after it sends
-                <x:ref>RST_STREAM</x:ref>.  <x:ref>PUSH_PROMISE</x:ref> causes a stream to become
-                "reserved" even if the associated stream has been reset.  Therefore, a
-                <x:ref>RST_STREAM</x:ref> is needed to close an unwanted promised stream.
+                A stream can transition from this state to "closed" by receiving either an
+                END_STREAM flag or a <x:ref>RST_STREAM</x:ref> frame.
+              </t>
+            </x:lt>
+            <x:lt hangText="closed:">
+              <t>
+                <vspace blankLines="0"/>
+                The "closed" state is the terminal state.
+              </t>
+              <t>
+                An endpoint MUST NOT send frames other than <x:ref>PRIORITY</x:ref> on a closed
+                stream.  An endpoint that receives any frame other than <x:ref>PRIORITY</x:ref>
+                or <x:ref>RST_STREAM</x:ref> in this state MUST treat that as a 
+		<xref target="StreamErrorHandler">stream error</xref> of type
+                <x:ref>STREAM_CLOSED</x:ref>.  
+              </t>
+              <t>
+                <x:ref>PRIORITY</x:ref> frames can be sent on closed streams to prioritize streams

```

Comment:
Probably, but I noticed that it is other states are not consistenty quoted, so was following the practise of what was before me.... but will change to consistency as I think that is best.